docs/docsource/Installation/topics/adding_NFS_storage.rst

```diff
@@ -38,76 +38,78 @@ and their default values.
-   | ``persistentVolume.readOnly``      | If true, persistent volume will be    | ``false``                    |
-   |                                    | read-only                             |                              |
-   +------------------------------------+---------------------------------------+------------------------------+
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | Parameter                             | Description                           | Default                      |
+   +=======================================+=======================================+==============================+
+   | ``pfsd.name``                         | Name of the pfsd component            | ``pfsd``                     |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.replicaCount``                 | Number of pfsd replicas               | ``1``                        |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.image.repository``             | pfsd image repository                 | ``zenko/pfsd``               |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.image.tag``                    | pfsd image tag                        | ``0.1``                      |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.image.pullPolicy``             | pfsd image pull policy                | ``IfNotPresent``             |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.service.type``                 | pfsd service type                     | ``ClusterIP``                |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.service.port``                 | pfsd service port                     | ``80``                       |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.resources``                    | pfsd resource requests and limits     | ``{}``                       |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.nodeSelector``                 | Node labels for pfsd pod assignment   | ``{}``                       |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.tolerations``                  | Node taints to tolerate               | ``[[``                       |
```

Comment:
```suggestion
   | ``pfsd.tolerations``                  | Node taints to tolerate               | ``[]``                       |
```

Reply:
Yeah but I'll fix it. Thanks for the find!